sources/main.cpp

```diff
@@ -207,6 +207,27 @@ int main(int argc, char* argv[]) {
     ButtonHandler* buttonHandler = new ButtonHandler(hwFactory->getInterruptHandler(), yioapi);
     qmlRegisterSingletonType<ButtonHandler>("ButtonHandler", 1, 0, "ButtonHandler", &ButtonHandler::getQMLInstance);
 
+    // reset combination was pressed on startup
+    if (buttonHandler->resetButtonsPressed()) {
+        // create marker file
+        QFile file("/firstrun");
+        if (file.open(QIODevice::WriteOnly)) {
+            file.close();
+        }
+
+        // restore default config
+        if (QFile::exists("/boot/config.json")) {
+            QFile::remove("/boot/config.json");
+        }
+        QFile::copy("/opt/yio/app/config.json", "/boot/config.json");
```

Comment:
Don't use a hardcoded path. There's $YIO_APP_DIR holding the app directory.
In case the `$YIO_APP_DIR/config.json` file is missing don't remove/rename `/boot/config.json`, otherwise the app most likely won't start anymore!

Reply:
Using ENV variables now.